.gitpod.yml

```
@@ -0,0 +1,59 @@
+# Building pandas on init
+# Might delegate this later to prebuild with Q2 improvements on gitpod
+# https://www.gitpod.io/docs/config-start-tasks/#configuring-the-terminal
+# -------------------------------------------------------------------------
+
+# assuming we use dockerhub: name of the docker user, docker image, tag, e.g. https://hub.docker.com/r/pandas/pandas-gitpod/tags
+image:
+  file: pythonpandas/pandas-gitpod:latest
```

```suggestion
image: pythonpandas/pandas-gitpod:latest
```

On my fork I had to change it to this to make it working